float32 ms_error 
float64 force_desired
float32 force_event
float64 force_weiss
float32 force_robot
float32 force_weiss2
float64 delta_z
float32 force_AR_desired
float32 force_AR_applied
